lib/import-export-visitor.js

```diff
@@ -186,6 +210,13 @@ module.exports = class ImportExportVisitor extends Visitor {
       const bodyInfo = this.bodyInfos[i];
       const parts = [];
 
+      // We don't need to add a "use strict" directive unless the compiler
+      // made any changes.
+      if (this.madeChanges &&
+          bodyInfo.needToAddUseStrictDirective) {
+        parts.push('"use strict";');
```

Comment:
It's a property of the `importExportVisitor`, not the AST. There's a `_bodyInfoByName` property that we put on the parent node [here](https://github.com/benjamn/reify/blob/919b684f34898fd71e2b7d50ee9cd9190dab10f4/lib/import-export-visitor.js#L139-L142), which is differentiated by the leading `_`.